src/triage/cli.py

```diff
@@ -436,5 +437,32 @@ def history(self, args):
         db_history(dburl=self.root.db_url)
 
 
+@Triage.register
+class SavePredictions(Command):
+    """Save the predictions of selected model groups"""
```

Comment:
This just saves test set (not training set) predictions, right? It's pretty rare we use the training set predictions, but might be good to specify this in the docstring/usage docs

Reply:
Changed the name and specified that it's only the test predictions